apps/opik-backend/config.yml

```diff
@@ -88,3 +88,6 @@ metadata:
 
 cors:
   enabled: ${CORS:-false}
+
+encryption:
+  key: ${COMET_ENCRYPTION_KEY:-'GiTHubiLoVeYouAA'}
```

Comment:
```suggestion
  key: ${OPIK_ENCRYPTION_KEY:-'GiTHubiLoVeYouAA'}
```

Reply:
It was draft version:) And updated already.